### PR TITLE
[IMP] website: introduce `s_accordion_image` snippet

### PR DESCRIPTION
**Pull request by Brieuc-brd**
This commit adds the new `s_accordion_image` snippet.

task-4094398
Part of task-4077427

Requires:
- https://github.com/odoo/design-themes/pull/870

Capture d’écran 2024-08-20 à 14 04 12

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/s_accordion_image.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            <div class="row align-items-start">
+                <div class="col-lg-7 pt16">
+                    <h2 class="h3-fs">Top questions answered</h2>
+                    <p class="lead">In this section, you can address common questions efficiently.</p>
+                    <p><br/></p>
+                    <t t-call="website.s_accordion">
+                        <t t-set="extra_classes" t-valuef="accordion-flush"/>
+                    </t>
+                </div>
+                <div class="col-lg-5 pt16" data-name="Accordion Box">
```
Comment (anso-odoo) [inline addons/website/views/snippets/s_accordion_image.xml:16]: Not sure `data-name="Accordion Box"` is useful. I think that keeping default "Column" will do the job ?

**Review by anso-odoo — COMMENTED** [review]
Maybe we could use an offset to make it breath a bit more. WDYT ? 

Screenshot 2024-08-20 at 10 56 18

File: addons/website/views/snippets/s_accordion_image.xml (hunk 1 above)
Comment (Brieuc-brd) [reply addons/website/views/snippets/s_accordion_image.xml:16]: This was made to maintain consistency with `s_faq_collapse`:) 
Capture d’écran 2024-08-20 à 11 21 02

Comment (anso-odoo) [reply addons/website/views/snippets/s_accordion_image.xml:16]: Okay that's what I thought, it was not intended... You set it on the image column ;)

Comment (Brieuc-brd) [reply addons/website/views/snippets/s_accordion_image.xml]: Oops good catch ! Fixed ✅

**Review by anso-odoo — APPROVED** [review]
LGTM, thanks !

**Comment by bso-odoo** [comment]
@robodoo r+

**Pull request closed without merging**